odoo/orm/models.py

```diff
@@ -4259,8 +4280,14 @@ def create(self, vals_list: list[ValuesType]) -> Self:
                 if field.store:
                     stored[key] = val
                 if field.inherited:
+                    if not self._has_field_access(field, 'write'):
+                        # we cannot write the field, let it default when inheriting
+                        continue
                     inherited[field.related_field.model_name][key] = val
                 elif field.inverse and field not in precomputed:
+                    if not self._has_field_access(field, 'write'):
+                        # we cannot write the field, use the default value
+                        continue
```

Comment:
That's weird to ignore without notice, don't you think? Also the check is already done before right ?

Reply:
Here, you have checked values, but also new default values (which are unchecked).
If we got a default value from the system that we cannot write on the field, remove it. Otherwise writing it will fail in `determine_inverse`.